ex7.8_NumPy_replace_array.py

```diff
@@ -0,0 +1,26 @@
+import numpy
+import numpy as np
+
+
+def replace_number(matrix: numpy.ndarray, number: int, sign: chr):
+    """
+    function that replace all numbers in a given array which is equal, less and greater to a given number.
+    :param matrix: matrix
+    :param number: number
+    :param sign:
+    :return:  array which is equal, less and greater to a given number.
+    """
+    if sign == "=":
+        print(np.where(matrix == number, 100, matrix))
+    elif sign == "<":
+        print(np.where(matrix < number, 100, matrix))
+    elif sign == ">":
+        print(np.where(matrix > number, 100, matrix))
```

Comment:
Why are you printing and not returning?
You should always try and seperate output from logic.
https://wiki.c2.com/?SeparateIoFromCalculation